CBLClient/Apps/CBLTestServer-Dotnet/TestServer/EncryptionKeyMethods.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Net;
+
+using JetBrains.Annotations;
+
+using static Couchbase.Lite.Testing.DatabaseMethods;
+
+namespace Couchbase.Lite.Testing
+{
+    internal static class EncryptionKeyMethods
+    {
+        //public static void EncryptionCreate([NotNull] NameValueCollection args,
```

Comment:
Does this have to be uncommented for 2.1?

Reply:
we may might. also I'm not sure if the method is Ok with the new API. Hence I guess it would be better to commit as commented code.

Reply:
2.1 has encrypted feature. cbl20-ver1 should have uncommented code as I have tested and checked into cbl20-ver1? Do you see this commented?

Reply:
in cbl20-ver1, it is still commented.

Reply:
After discussing with Sri, we decided to uncomment this code.